sync/models/sync_project.py

```diff
@@ -44,7 +44,7 @@ class SyncProject(models.Model):
     name = fields.Char(
         "Name", help="e.g. Legacy Migration or eCommerce Synchronization", required=True
     )
-    active = fields.Boolean(default=True)
+    active = fields.Boolean(default=False)
```

Comment:
@yelizariev What do you think about it?

Reply:
I't good for module creating, but not for a custom Project created via UI